expo/tests/gbe/test_reports.py

```diff
@@ -37,15 +42,23 @@
 )
 from tests.factories.ticketing_factories import (
     TransactionFactory,
-    )
+)
+from tests.contexts import (
+    ActTechInfoContext,
+    ClassContext,
+    VolunteerContext,
+)
 import ticketing.models as tix
 
-import tests.functions.gbe_functions as functions
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    login_as,
+)
 
 
 def _create_scheduled_show_with_acts(conference=None, qty=6):
     if not conference:
-        conference = ConferenceFactory.create()
+        conference = ConferenceFactory()
```

Comment:
That was my fear too... obviously the unit tests wouldn't work if that was the case...